PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,83 @@
+<!--
+  If you have a relevant JIRA issue number, please put it in the issue title.
+  Example: MODQM-3 - Implement GET records-editor/marc-records/{id} endpoint
+
+  TL;DR
+    - https://www.youtube.com/watch?v=5aHmO_S8FQ4
+    - http://www.olitreadwell.com/2016/05/22/how-to-write-great-pull-requests/
+    - https://www.atlassian.com/blog/git/written-unwritten-guide-pull-requests
+-->
+
+## Purpose
+<!--
+  Why are you making this change? There is nothing more important
+  to provide to the reviewer and to future readers than the cause
+  that gave rise to this pull request. Be careful to avoid circular
+  statements like "the purpose is to update the schema." and
+  instead provide an explanation like "there is more data to be provided and stored for Purchase Orders 
+  which is currently missing in the schema"
+
+  The purpose may seem self-evident to you now, but the standard to
+  hold yourself to should be "can a developer parachuting into this
+  project reconstruct the necessary context merely by reading this
+  section."
+
+  If you have a relevant JIRA issue, add a link directly to the issue URL here.
+  Example: https://issues.folio.org/browse/MODQM-3
+ -->
+
+## Approach
+<!--
+ How does this change fulfill the purpose? It's best to talk
+ high-level strategy and avoid code-splaining the commit history.
+
+ The goal is not only to explain what you did, but help other
+ developers *work* with your solution in the future.
+-->
+
+#### TODOS and Open Questions
+<!-- OPTIONAL
+- [ ] Use GitHub checklists. When solved, check the box and explain the answer.
+-->
+- [ ] Check logging
```

Comment:
that was inside of the comment in most pr template I have seen.

Reply:
are you suggesting to remove this line ?

Reply:
no, move to inside of comment
<!-- OPTIONAL
- [ ] Use GitHub checklists. When solved, check the box and explain the answer.
- [ ] Check logging
-->

Reply:
oh ok will do thanks